### ISSUE TITLE
extend_phi_access does not work with has_many relationships

### ISSUE THREAD

**Issue by ncallaway** [apsislabs/phi_attrs#9]
Calling `extend_phi_access` does not adequately work for relationships that were defined with `has_many` instead of belongs to.

**Example:**

```ruby
class MyAwesomeRecord < ApplicationRecord
  phi_model

  belongs_to :awesome_record_meta
  has_many :user_fields

  extend_phi_access :awesome_record_meta, :user_fields
end

...later

awesome = MyAwesomeRecord.first
awesome.allow_phi!("user", "context")

awesome.awesome_record_meta.meta_field # OK, works as expected
awesome.user_fields.first.user_field # Explodes with PHIAccessException
```

Fix will likely need to be in the `phi_extend_access` method in https://github.com/apsislabs/phi_attrs/blob/master/lib/phi_attrs/phi_record.rb#L286

**Comment by HenryKeiter** [comment]
This should probably be implemented using Chris' multi-target allows: https://github.com/apsislabs/phi_attrs/pull/8

**Comment by HenryKeiter** [comment]
This is fixed in the current version of https://github.com/apsislabs/phi_attrs/pull/8 though that PR may not end up being merged as-is.

**Comment by HenryKeiter** [comment]
Resolved in #8
